2048 machine coding problem/two_zero_four_eight_driver.java

```diff
@@ -0,0 +1,25 @@
+package TwoZeroFourEight;
+
+import java.util.Scanner;
+
+public class two_zero_four_eight_driver {
```

Comment:
Class names should not have _. It should be camelCase with first letter Capital as well.